data/building-hours/5-2-asc.yaml

```diff
@@ -1,9 +1,9 @@
-name: 'Academic Support Center'
-abbreviation: 'ASC'
-category: 'Help and Support'
+name: ASC
+subtitle: Academic Support Center
```

Comment:
This was flipped to match all the other abbreviations. All the other ones use the way I changed it too.